victron_ble/devices/solar_charger.py

```diff
@@ -0,0 +1,90 @@
+from enum import Enum
+from typing import Any, Dict
+
+from construct import Int16sl, Int16ul, Struct
+
+from victron_ble.devices.base import Device, DeviceData
+
+
+class ChargeState(Enum):
+    OFF = 0
+    BULK = 3
+    ABSORPTION = 4
+    FLOAT = 5
+
+
+class SolarChargerData(DeviceData):
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self._data = data
+
+    def get_charge_state(self) -> ChargeState:
+        """
+        Return an enum indicating the current meter type
+        """
+        return self._data["charge_state"]
+
+    def get_voltage(self) -> float:
+        """
+        Return the voltage in volts
```

Comment:
Can you update the comment to indicate if this is battery voltage or the solar input voltage?

Reply:
I'd be tempted to rename the method to `get_battery_voltage`